### PR TITLE
cisco merakai c2c app doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/cisco-meraki-c2c.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('/img/integrations/security-threat-detection/ciscomeraki.png')} alt="ciscomeraki-logo" width="75"/>
 
-The Sumo Logic app for Cisco Meraki provides real-time insights into the events and helps you to identify potential network events along with admin activities. This app can effectively manage and optimize your network performance, enhance security, and proactively respond to potential threats. The comprehensive insights and monitoring capabilities enable efficient network administration and contribute to a robust and secure network infrastructure.
+The Sumo Logic app for Cisco Meraki C2C provides real-time insights into the events and helps you to identify potential network events along with admin activities. This app can effectively manage and optimize your network performance, enhance security, and proactively respond to potential threats. The comprehensive insights and monitoring capabilities enable efficient network administration and contribute to a robust and secure network infrastructure.
```
Comment (JainM6) [inline docs/integrations/saas-cloud/cisco-meraki-c2c.md:11]:
```suggestion
The Sumo Logic app for Cisco Meraki provides real-time insights into the events and helps you to identify potential network events along with admin activities. This app can effectively manage and optimize your network performance, enhance security, and proactively respond to potential threats. The comprehensive insights and monitoring capabilities enable efficient network administration and contribute to a robust and secure network infrastructure.
```

Comment (JainM6) [reply docs/integrations/saas-cloud/cisco-meraki-c2c.md:11]: We should link Cisco Meraki C2C here.

**Review by JainM6 — CHANGES_REQUESTED** [review]
Minor change

File: docs/integrations/saas-cloud/cisco-meraki-c2c.md (hunk 1 above)
Comment (JV0812) [reply docs/integrations/saas-cloud/cisco-meraki-c2c.md:11]: @JainM6 The app name itself is Cisco Meraki C2C. I had a discussion with dev guys and they wanted the app name in this way just to differentiate this app from the existing Cisco Meraki (uses Syslog source) and this app uses Sumo Logic C2C source.

File: docs/integrations/saas-cloud/cisco-meraki-c2c.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+```sql title="Total Activities"
+_sourceCategory=cm_con2006 wiredMacs
+| json "ssid", "channels", "firstSeen", "lastSeen", "wiredMacs", "wiredVlans", "wiredLastSeen","bssids[*].bssid","bssids[*].detectedBy[*].device","bssids[*].detectedBy[*].rssi" as ssid, channels, first_seen, last_seen, wired_macs, wired_vlans, wired_last_seen,bssids,devices,rssi_values nodrop
+| count
+```
+
+## Set up collection
+
+To set up [Cloud to Cloud Integration Cisco Meraki Source](/docs/integrations/security-threat-detection/cisco-meraki/) for the Cisco Meraki - C2C app, follow the instructions provided. These instructions will guide you through the process of creating a source using the Cisco Meraki source category, which you will need to use when installing the app. By following these steps, you can ensure that your Cisco Meraki - C2C app is properly integrated and configured to collect and analyze your Cisco Meraki data.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/cisco-meraki-c2c.md:77]:
```suggestion
To set up data ingestion for the Cisco Meraki - C2C app, follow the instructions provided at [Cisco Meraki Source](/docs/integrations/security-threat-detection/cisco-meraki/). These instructions will guide you through the process of creating a source using the Cisco Meraki source category, which you will need to use when installing the app. By following these steps, you can ensure that your Cisco Meraki - C2C app is properly integrated and configured to collect and analyze your Cisco Meraki data.
```

File: docs/integrations/saas-cloud/cisco-meraki-c2c.md (hunk 1 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/cisco-meraki-c2c.md:11]: @JV0812 I had a follow-up question. Should it be `Cisco Meraki - C2C` or `Cisco Meraki C2C`. I see both formats in this doc.

**Review by kimsauce — CHANGES_REQUESTED** [review]
Two comments

File: docs/integrations/saas-cloud/cisco-meraki-c2c.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+- **Air Marshal Security Overview**. Provide wireless intrusion detection and prevention by monitoring security status and identifying potential vulnerabilities in the network.
+- **Enhanced Security Measures**. Monitor blocked connections, file scans, and malicious files to ensure a secure network environment. Prioritize your security efforts by identifying top clients and destinations based on security events and take proactive steps to protect the network and mitigate potential threats.
+- **Visualization and Reporting**. Provides visual representations of network data, making it easy for you to understand and interpret key metrics. Generate reports and share insights with your stakeholders to facilitate decision-making and drive network improvements.
+
+## Log types
+
+This app uses [Cisco Meraki source](/docs/integrations/security-threat-detection/cisco-meraki/) to collect the data from Cisco Meraki.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/cisco-meraki-c2c.md:25]: This URL points to another app doc, not a source. Did you mean ... 

```suggestion
This app uses [Cisco Meraki source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-meraki-source) to collect the data from Cisco Meraki.
```

**Review by kimsauce — DISMISSED** [review]
Minor fix in two places

**Review by kimsauce — APPROVED** [review]

**Pull request merged**